Doppler.BillingUser/Controllers/BillingController.cs

```diff
@@ -0,0 +1,39 @@
+using Doppler.BillingUser.DopplerSecurity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Doppler.BillingUser.Controllers
+{
+    [Authorize]
+    [ApiController]
+    public class BillingController
+    {
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpGet("/accounts/{accountname}/billing-information")]
+        public string GetBillingInformation(string accountname)
+        {
+            return $"Hello! \"you\" that have access to GetCurrentBillingInformation with accountname '{accountname}'";
+        }
+
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpPut("/accounts/{accountname}/billing-information")]
+        public string UpdateBillingInformation(string accountname)
+        {
+            return $"Hello! \"you\" that have access to UpdateBillingInformation with accountname '{accountname}'";
+        }
+
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpGet("/accounts/{accountname}/payment-methods/current")]
+        public string GetCurrentPaymentMethod(string accountname)
+        {
+            return $"Hello! \"you\" that have access to GetCurrentPaymentMethod with accountname '{accountname}'";
+        }
+
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpPost("/accounts/{accountname}/upgrade")]
```

Comment:
Perdón, reabriendo la discución de https://github.com/FromDoppler/doppler-billing-user/pull/4#discussion_r668004062, debería este recurso llamarse `upgrades`? quiero decir, creamos un nuevo `upgrade` con el POST?

```suggestion
        [HttpPost("/accounts/{accountname}/upgrades")]
```

Reply:
Con este POST creamos un nuevo `BillingCredit`, que implica el cambio de plan concreto, y el pago por dicho cambio. No tenemos una entidad Upgrade en si mismo, pero digamos que el BillingCredit en estos casos representaria eso mismo. 
Para confirmar si entiendo tu idea, crees que podria ir mejor `upgrades `porque cada vez que usamos este recurso se crearia uno nuevo?

Reply:
> Para confirmar si entiendo tu idea, crees que podria ir mejor upgrades porque cada vez que usamos este recurso se crearia uno nuevo?

Correcto.

[Abro chat](https://makingsense.slack.com/archives/CRXR62TC2/p1626178577334600) para discutir el nombre, por ahí encontramos algo mejor.